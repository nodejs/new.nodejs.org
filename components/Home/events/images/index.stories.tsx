import EventsImages from '.';
import type { Meta as MetaObj, StoryObj } from '@storybook/react';

type Story = StoryObj<typeof EventsImages>;
type Meta = MetaObj<typeof EventsImages>;

export const Default: Story = {};

export default { component: EventsImages } as Meta;
