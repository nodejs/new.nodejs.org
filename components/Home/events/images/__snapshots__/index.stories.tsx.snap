// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home/events/images Default smoke-test 1`] = `
<div class="images_image__dY6kn">
  <img src="/static/images/events/openjs-world-2023/teams.jpeg"
       alt="Event illustration image"
  >
</div>
`;
